server/store/datastore/migration/032_registries_add_user.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2024 Woodpecker Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package migration
+
+import (
+	"src.techknowlogick.com/xormigrate"
+	"xorm.io/xorm"
+)
+
+// Registry represents a docker registry with credentials.
+type RegistryV032 struct {
+	ID       int64  `json:"id"       xorm:"pk autoincr 'id'"`
+	OrgID    int64  `json:"org_id"   xorm:"NOT NULL DEFAULT 0 UNIQUE(s) INDEX 'org_id'"`
```

Comment:
Shouldn't this be the old version without OrgID?

Reply:
No, it does creates new table version

Reply:
AFAIK normally we sync the old state of the model in migrations to ensure the are synced. Then the migration is executed and afterwards the new model is syned in https://github.com/woodpecker-ci/woodpecker/blob/e5f3e67bf282bd938b5ab39295f738e2fefa7d4a/server/store/datastore/migration/migration.go#L119

Reply:
seems hard to catch errors this way to be sure it's working when upgrading from different versions but ok, removed sync as I do not depend on it that much